docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -15,22 +15,21 @@
     specific language governing permissions and limitations
     under the License.
 
-Extra Packages Reference
-''''''''''''''''''''''''
+Reference for package extras
```

Comment:
Or simply "Package Extras Explained" if audience includes end-users. "Reference" is correct but slightly technical term.

Reply:
We use that term in other parts (for example https://github.com/apache/airflow/blob/master/docs/apache-airflow/operators-and-hooks-ref.rst) so I prefer to keep it consistent :)